www/app/app.js

```diff
@@ -439,14 +439,19 @@ define(['angularAMD', 'app.routes', 'app.constants', 'app.notifications', 'app.p
 
 						$rootScope.MakeGlobalConfig();
 
-						if (typeof data.result.templates != 'undefined') {
+						if ( (typeof data.result.templates != 'undefined') || (typeof data.result.urls != 'undefined') ) {
```

Comment:
Maybe it's best to make a second IF block only for the urls (when undefined) ?

```
if (typeof data.result.urls != 'undefined')  {
...
}

```

Reply:
@gizmocuz   I am not sure to understand what you mean: this would require to duplicate the code to properly set the $rootScope.config.EnableTabCustom?

Reply:
I mean two seperate $.each loops depending if the result templates  or urls is undefined

```
if (typeof data.result.templates != 'undefined') {
	$.each(data.result.templates, function (i, item) {
		var cFile = item;
		var cName = cFile.charAt(0).toUpperCase() + cFile.slice(1);
		var cURL = "templates/" + cFile;
		customHTML += '<li><a href="javascript:SwitchLayout(\'' + cURL + '\')">' + cName + '</a></li>';
	});
}
```

```
if (typeof data.result.urls != 'undefined')  {
	$.each(data.result.urls, function (name, url) {
		var cName = name.charAt(0).toUpperCase() + name.slice(1);
		var cURL = url;
		customHTML += '<li><a target="_blank" href="' + cURL + '">' + cName + '</a></li>';
	});
}
```

Reply:
@gizmocuz ok sorry I missed that, even surprising it didn't crash...  changes done.